### PR TITLE
Optimise approach for displaying placeholders

### LINKED ISSUE #2708: Optimise approach for loading custom placeholders
The current apprach has to loop across the `custom_placeholders` dict everytime to get a placeholder image.

### PR DESCRIPTION
**Pull request by shubham-padia**
@niranjan94 @SaptakS 
Resolves #2708 
selection_025

### CODE REVIEW

File: app/templates/gentelella/guest/event/base.html [hunk 1]
```diff
@@ -89,10 +73,12 @@
 {% endblock %}
 
 {% block title %}
+    {{ event.sub_topic }}
     {{ event.name }}
 {% endblock %}
 
 {% block body %}
+
```
Comment (SaptakS) [inline app/templates/gentelella/guest/event/base.html]: Please remove these extra lines.

File: app/templates/gentelella/guest/event/base.html [hunk 2]
```diff
@@ -178,6 +164,7 @@ <h4 style="font-weight: 300;"><i class="glyphicon glyphicon-map-marker"></i> {{
         </div>
         <div class="col-md-2 col-md-push-1 hidden-xs hidden-sm">
             <div>
+
```
Comment (SaptakS) [inline app/templates/gentelella/guest/event/base.html]: Also these

Comment (shubham-padia) [inline app/templates/gentelella/guest/event/base.html]: Done as requested @SaptakS

**Comment by shubham-padia** [comment]
@SaptakS Removed the anamoly.

**Comment by shubham-padia** [comment]
@SaptakS @niranjan94 can you please review again. Made the required changes :smiley:

**Pull request merged**